tc/sgx/trusted_worker_manager/enclave/work_order_processor.cpp

```diff
@@ -61,10 +61,12 @@ namespace tcf {
             "payloadFormat",
             "invalid request; failed to retrieve payload format");
 
+        // verifyingKey is optional field. Hence don't throw exception
```

Comment:
For better clarity, this could be renamed as requesterVerifyingKey. Also add a comment this parameter is not mentioned in spec and added to verify requesterSignature

Reply:
Ok, i will add a comment